source/common/common/block_memory_hash_set.h

```diff
@@ -292,7 +292,7 @@ template <class Value> class BlockMemoryHashSet : public Logger::Loggable<Logger
   struct Control {
     BlockMemoryHashSetOptions options; // Options established at map construction time.
     uint64_t hash_signature;           // Hash of a constant signature string.
-    uint32_t num_bytes;                // Bytes allocated on behalf of the map.
+    size_t num_bytes;                  // Bytes allocated on behalf of the map.
```

Comment:
nit: can we use uint64_t here and elsewhere to be super clear about the sizes?

Reply:
ok; done